### ISSUE TITLE
Async does not pass options to processValidationResults

### ISSUE THREAD

**Issue by kirill-zhirnov** [ansman/validate.js#19]
If you install validate.js from npm (npm install validate.js):
validate.js, line: 133
```javascript
var errors = v.processValidationResults(results);
```

In GIT repository:
validate.js, line: 133
```javascript
var errors = v.processValidationResults(results, options);
```

Please, re-publish NPM package.

Thank you!

**Comment by ansman** [comment]
This works as expected for me:
```
$ npm install validate.js
validate.js@0.3.2 node_modules/validate.js
```

Which version are you trying to install?

**Comment by kirill-zhirnov** [comment]
Exactly:
```
validate.js@0.3.2 node_modules/validate.js
```

But it has the issue:

```javascript
var validate = require('validate.js');

var constraints = {
  username: {
    presence: true
  }
};

validate.async({}, constraints, {
  fullMessages: false
}).then(function(result) {
  console.log('Success', result);
}).fail(function(fail) {
  console.log('Errors', fail);
}).done();
```

The output will be:
```
Errors { username: [ 'Username can\'t be blank' ] }
```

But expected:
```
Errors { username: [ 'can\'t be blank' ] }
```

**Comment by Jokero** [comment]
@kirill-zhirnov You can install package from master branch

**Comment by kirill-zhirnov** [comment]
I can :) But it is a bug :)

**Comment by Jokero** [comment]
Waiting for the next release)

**Comment by ansman** [comment]
Fixed in 88747e0d13ececddd4141591e29b231c869d72b9

**Comment by ansman** [comment]
Validate 0.4.0 is out now
